<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayListForm</class>
 <widget class="QWidget" name="PlayListForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>16</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>571</width>
     <height>471</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="play_list">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QLabel" name="label_PL_back">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>567</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(45, 47, 51);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_PL_conut">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>6</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(130, 131, 133);</string>
     </property>
     <property name="text">
      <string>总100首</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_PL_shoucang">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>5</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>收藏全部</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_PL_delete">
     <property name="geometry">
      <rect>
       <x>509</x>
       <y>5</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_PL_conut_2">
     <property name="geometry">
      <rect>
       <x>498</x>
       <y>5</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(130, 131, 133);</string>
     </property>
     <property name="text">
      <string>|</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_playlist">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>567</width>
       <height>411</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="history_list">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QLabel" name="label_PL_back_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>567</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(45, 47, 51);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_PL_delete_2">
     <property name="geometry">
      <rect>
       <x>509</x>
       <y>5</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_PL_conut_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>6</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(130, 131, 133);</string>
     </property>
     <property name="text">
      <string>总100首</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_historylist">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>567</width>
       <height>411</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <zorder>tabWidget</zorder>
  <zorder>pushButton_close</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
